assets/src/util/math.ts

```diff
@@ -9,46 +7,3 @@ import { LatLng, LatLngLiteral } from "leaflet"
  */
 export const clamp = (value: number, min: number, max: number): number =>
   Math.min(Math.max(min, value), max)
-
-interface ClosestPosition {
-  position: LatLngLiteral
-  index: number
-  distance: number
-}
-
-/**
- * Finds the element in {@link positions} that is closest to {@link point}
- * @param positions List of coordinates to search
- * @param point Point of reference to check distance to
- * @returns
- * if {@link positions} is empty, returns `undefined`
- *
- * Otherwise, returns a {@link ClosestPosition} object containing
- * - {@link ClosestPosition.position}: the closest element in {@link positions} to {@link point}
- * - {@link ClosestPosition.index}: the index of the element in {@link positions}
- * - {@link ClosestPosition.distance}: the distance to {@link point}
- */
-export const closestPosition = (
```

Comment:
I'm on the fence about actually removing this, but the other part of this PR removes the only invocation of this function, so it's not strictly needed. Part of me thinks that we might keep it around "just in case", and I also worry that that's how code-hoarding happens.